src/web/lib/components/AppHeader/index.js

```diff
@@ -7,7 +7,6 @@ import {
 } from "../../../../lib/generators";
 
 import { version } from "../../../../../package.json";
```

Comment:
Looks like this needs to be snipped out too, since the survey link on line 39 was its only use.